src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -63,6 +63,19 @@
             </valueSetDefinition>
         </valueSet>
     </fields>
+    <fields>
+        <fullName>CommitmentId__c</fullName>
```

Comment:
@sam-knox Would you mind reviewing the new field label, help text, etc. just to make sure it all looks good?

Reply:
@force2b - We decided to go with unique = true?